src/Symfony/Bundle/FrameworkBundle/Resources/config/annotations.xml

```diff
@@ -36,7 +36,7 @@
         <service id="annotations.cache_warmer" class="Symfony\Bundle\FrameworkBundle\CacheWarmer\AnnotationsCacheWarmer">
             <argument type="service" id="annotations.reader" />
             <argument>%kernel.cache_dir%/annotations.php</argument>
-            <argument>#^Symfony\\(?:Component\\HttpKernel\\|Bundle\\FrameworkBundle\\Controller\\(?!AbstractController$|Controller$))#</argument>
+            <argument>#^Symfony\\(?:Component\\HttpKernel\\|Bundle\\FrameworkBundle\\Controller\\(?!AbstractController$|RedirectController$|Controller$))#</argument>
```

Comment:
let's add the TemplateController also?

Reply:
I also thought about which controllers have to be added additionally.

So the regex could be changed to `(?![A-Za-z]*Controller$)` to catch them all

Reply:
I'm not sure negative assertions support variable-length matches. Please double-check if you want to follow this path.

Reply:
See https://3v4l.org/2FKNp

This would work

Reply:
ok good, let's update to `(?!.*Controller$)` then